### PR TITLE
Updating ontology load json

### LINKED ISSUE #21: Making SciSpacy compatible with the new Ontology.py
-Merging the features of the previous KnowledgeBase class with the common Ontology.py object
-Updating file_cache.py to make it more user firendly.
-Creating a new class for Scispacy's embeddings to separate features exclusive to Ontology.py and features exclusive to SciSpacy
-Updating the rest of the Linker to make it work.

### PR DESCRIPTION
**Pull request by batuinmetz**

### CODE REVIEW

File: bioel/tests/test_ontology.py [hunk 1]
```diff
@@ -26,6 +26,23 @@ def test_medic_loader(self):
             self.check_multiprefix(ontology)
             self.check_unique_cui(ontology)
 
+    def test_json_loader(self):
+        """
+        TestCase - 1: Read from the Json/Jsonl Ontology Directory and check for data formatting issues in the entities.
```
Comment (prasbathala) [inline bioel/tests/test_ontology.py:33]: Can you add some more test cases for the JSON.
1. Give a local path.
2. Give a URL to the dataset.
3. Check if it is already downloaded, and it is accessing from the local cache or not. 
4. check if not downloaded, is it able to download the dataset.

File: bioel/bioel/ontology.py [hunk 2]
```diff
@@ -392,9 +396,44 @@ def load_ncbi_taxon(self):
     def load_csv(self):
         pass
 
-    def load_json(self):
-        pass
+    @classmethod
+    def load_json(cls, filepath, name=None):
```
Comment (prasbathala) [inline bioel/bioel/ontology.py:490]: Can you also write a detailed description of what functionalities `load_json` can offer/support?

**Review by prasbathala — COMMENTED** [review]
Overall looks good. Need to test on different functionalities. Once all the unit test cases are in, It can be merged.

**Review by prasbathala — CHANGES_REQUESTED** [review]
Requesting changes for test cases.

**Review by batuinmetz — COMMENTED** [review]
done

**Pull request merged**